docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,172 @@
+| **rules.mutators**               |  **NO**   | Specifies the list of the request mutators.                                                                                                                                                                                                                                                                                                  | Currently, the `Headers` and `Cookie` mutators are supported. For more information, see the [documentation](https://github.com/kyma-project/api-gateway/blob/main/docs/user/custom-resources/apirule/04-40-apirule-mutators.md). |
+| **rules.noAuth**                 |  **NO**   | Setting `noAuth` to `true` disables authorization.                                                                                                                                                                                                                                                                                           | When `noAuth` is set to true, there cannot be any access strategy defined                                                                                                                                                        |
+| **rules.accessStrategy**         |  **NO**   | Specifies the access strategy.                                                                                                                                                                                                                                                                                                               | Supported are `extAuth` or `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                               |
+| **rules.accessStrategy.extAuth** |  **NO**   | Specifies the list of external authorizers. For more information see the [External Authorizer ADR](https://github.com/kyma-project/api-gateway/issues/938).                                                                                                                                                                                  |                                                                                                                                                                                                                                  |
```

Comment:
can we prepare a page or section for `extAuth` providing the documentation / spec similar to this one? since now it is fully typed, we should specify full spec for the v1beta2.